.github/workflows/deploy.yml

```diff
@@ -78,6 +78,9 @@ jobs:
       - name: Build demo & web components
         run: npm run build:demo
 
+      - name: Build docs
+        run: npm run docs:build -- --base=/geonetwork-ui/${{env.BRANCH_NAME}}/docs/ && mkdir -p dist/docs && mv docs/.vitepress/dist/* dist/docs
```

Comment:
Maybe add a link in the PR comment below to easily access the doc on the branch?